log_injection/Rahul-Sahani04_MikoChan-Telegram-Bot-V2/V2.py

```
@@ -0,0 +1,128 @@
+    HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT: HarmBlockThreshold.BLOCK_NONE,
+    HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT: HarmBlockThreshold.BLOCK_NONE,
+}
+
+model = genai.GenerativeModel("gemini-1.5-flash", safety_settings=safety_settings)
+history_file_path = "MikoChanHistory/MikoChan.pkl"
+
+# Ensure the history directory exists
+os.makedirs(os.path.dirname(history_file_path), exist_ok=True)
+
+if os.path.exists(history_file_path):
+    person = pkl.load(open(history_file_path, "rb"))
```

Ruff (0.13.3)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)